src/pages/PeerDiscussionPage/peer.js

```diff
@@ -0,0 +1,146 @@
+      onPress: () => console.log("Interview your friend"),
+    },
+    {
+      id: 3,
+      text: "Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document",
+      onPress: () => console.log("Contests and Statistics Dashboard"),
+    },
+    {
+      id: 4,
+      text: "Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document",
+      onPress: () => console.log("Classroom"),
+    },
+  ];
+
+//   myChangeHandler = (event) => {
+//     this.setState({username: event.target.value});
+//   }
+
+  return (
+    <Container>
+      <Box className={classes.profiletitle}>
+        <Typography variant="h2">Discussion Form</Typography>
+      </Box>
+      <Box>
+        <Typography className={classes.profiletitle} variant="h5">Discussion Topic</Typography>
+      </Box>
+      <Divider />
+      <Box style={{padding: "10px"}}>
+        <Typography variant="p">Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document .</Typography>
+      </Box>
+      <Box>
+        <Typography className={classes.profiletitle} variant="h5">Responses </Typography>
+      </Box>
+      <Divider />
+      
+      <div className={classes.margin}>
+        <Grid container spacing={1} alignItems="flex-end">
+          <Grid item>
+          <AccountCircleOutlined style={{height: "40px"}}/>
+          </Grid>
+          <Grid item>
+            <TextField id="outlined-basic" label="Type a Response" variant="outlined" style={{width: "150%"}}/>
+          </Grid>
+          <Grid item>
+            <Button style={{marginLeft: "50px"}}><ArrowForwardIosIcon /></Button>
+          </Grid>
+        </Grid>
+      </div>
+      <Divider />
+
+      <div className={classes.margin}>
+        <Grid container spacing={1} alignItems="flex-end">
+          <Grid item>
+            <AccountCircleOutlined style={{height: "40px"}}/>
+          </Grid>
+          <Grid item>
+            <Typography variant="p" style={{display: 'inline-block'}}>Lorem ( 27/03/2021 ) </Typography>
+            {/* <TextField id="outlined-basic" label="Type a Response" variant="outlined" style={{width: "340px"}} /> */}
+          </Grid>
+          <Box style={{padding: "10px"}}>
+                <Typography variant="p">Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content.</Typography>
+            </Box>
+        </Grid>
+      </div>
+      <Divider />
+
+      <div className={classes.margin}>
+        <Grid container spacing={1} alignItems="flex-end">
+          <Grid item>
+            <AccountCircleOutlined style={{height: "40px"}}/>
+          </Grid>
+          <Grid item>
+            <Typography variant="p" style={{display: 'inline-block'}}>Ipsum ( 26/03/2021 )</Typography>
+            {/* <TextField id="outlined-basic" label="Type a Response" variant="outlined" style={{width: "340px"}} /> */}
```

Comment:
Hey, @poojadas07 there are some conflicts in your PR please resolve that and  you can use  `InputProps={{endAdornment: <Button style={{ marginLeft: "50px" }}>
              <ArrowForwardIosIcon />
            </Button>}}` in the Textfield to make button center to it and you can also give `10px Left padding` to other topics list arrow icon.

Reply:
I see that you have completely deleted the `package-lock.json` file  I think that should not be added to your PR!
and @poojadas07 try to make a different branch for different issues. I see you have committed the local changes to your forked main branch this approach will create issues for you. 

Rest everything is great from my side :)

Reply:
@Abhishek-kumar09  ping!

Reply:
Yeahhh...I am getting some issues. Can you please help me what should I do now.

Reply:
Hey, @poojadas07  have you added the upstream branch to the forked repo!

Reply:
In your CLI is it showing merge conflict message!

Reply:
> Hey, @poojadas07 have you added the upstream branch to the forked repo!

yes

Reply:
No merge conflicts.....but still I am not able to push my code .

Reply:
go for this `git merge --abort` command

Reply:
Showing ----fatal: There is no merge to abort (MERGE_HEAD missing).

Reply:
Actually it is because of this 

```
src/Routes.js
Syntax error: Identifier 'Container' has already been declared (17:10)

  15 | import About from "./pages/Info/About";
  16 | import PeerPage from "./pages/PeerDiscussionPage/peer";
> 17 | import { Container } from "@material-ui/core";
     |          ^
  18 |
  19 | const renderRoutes = () => (
  20 |   <Container maxWidth="xs" style={{ margin: "auto", padding: 0 }}>

```

Reply:
please remove that extra import statement from your local repo!

Reply:
yess...I did

Reply:
hey @poojadas07  please don't delete your conversations 😊 as this will help other students who are new or beginners.